### ISSUE TITLE
Feature request: RKWard support

### ISSUE THREAD

**Issue by lstrtstr** [ArcadeAntics/this.path#19]
Unfortunately, this package doesn't work in RKWard. It would be mighty neat if it worked in RKWard. (None of the alternatives work in RKWard either.)

(I have no idea how to implement RKWard support.)

**Comment by ArcadeAntics** [comment]
Hiya @lstrtstr !


I would be happy to implement `package:this.path` to work with the GUI [RKWard](https://github.com/KDE/rkward). I'm having trouble using it right now, it installs correctly but then throws a strange error:

```
R engine has died - RKWard

Connection closed unexpectedly. Last error was:
Unknown error
```

So it may be a while before I've made any progress, but I'll let you know when I do.

**Comment by unDocUMeantIt** [comment]
please let us know how you installed RKWard. looks like a mismatch between the R version it was compiled for and the one it tries to use. version 0.8.0 is around the corner, you could try one of the [CI builds](https://cdn.kde.org/ci-builds/education/rkward/master/) for your operating system. it now let's you choose between found R installations.

**Comment by ArcadeAntics** [comment]
Hiya @unDocUMeantIt !


Are you one of the RKWard devs?

Either way, I resolved the issue, although I don't like the solution.

I previously defined this environment variable:

`R_DEFAULT_PACKAGES=methods,datasets,utils,grDevices,graphics,stats,essentials,this.path`

and with it defined, RKWard threw the error as previously shown. When I undefine it, the R session starts as normal.

I'll open an issue and link to here.

**Comment by unDocUMeantIt** [comment]
> Are you one of the `RKWard` devs?

yes, i'm part of the team (mostly R related stuff, not so much Qt).

> Either way, I resolved the issue, although I don't like the solution.
> 
> I previously defined this environment variable:
> 
> `R_DEFAULT_PACKAGES=methods,datasets,utils,grDevices,graphics,stats,essentials,this.path`
> 
> and with it defined, `RKWard` threw the error as previously shown. When I undefine it, the R session starts as normal.

interesting, i can't replicate the issue with the AppImage (seems the variable is simply ignored), but we'll sure look into it if you report it.

> I'll open an issue and link to here.

thanks! we're looking forward to getting `this.path` compatible with RKWard :)

**Comment by waynelapierre** [comment]
I also would like to see RKWard support!
